components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/hardware-wallet-connect/accounts-list.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react'
+import { create } from 'ethereum-blockies'
+import { Checkbox, Select } from 'brave-ui/components'
+import {
+  ButtonsContainer,
+  DisclaimerWrapper,
+  HardwareWalletAccountCircle,
+  HardwareWalletAccountListItem,
+  HardwareWalletAccountListItemColumn,
+  HardwareWalletAccountsList,
+  SelectWrapper
+} from './style'
+import {
+  HardwareWallet,
+  HardwareWalletAccount,
+  HardwareWalletDerivationPathLocaleMapping,
+  HardwareWalletDerivationPathsMapping
+} from './types'
+import { reduceAddress } from '../../../../../utils/reduce-address'
+import locale from '../../../../../constants/locale'
+import { NavButton } from '../../../../extension'
+import { SearchBar } from '../../../../shared'
+import { DisclaimerText } from '../style'
+
+interface Props {
+  hardwareWallet: HardwareWallet
+  accounts: Array<HardwareWalletAccount>
+  onLoadMore: () => void
+  selectedDerivationPaths: string[]
+  setSelectedDerivationPaths: (paths: string[]) => void
+  selectedDerivationScheme: string
+  setSelectedDerivationScheme: (scheme: string) => void
+}
+
+export default function (props: Props) {
+  const {
+    hardwareWallet,
+    selectedDerivationScheme,
+    setSelectedDerivationScheme
+  } = props
+  const derivationPathsEnum =
+    HardwareWalletDerivationPathsMapping[hardwareWallet]
+
+  return (
+    <>
+      <SelectWrapper>
+        <Select
+          value={selectedDerivationScheme}
+          onChange={(e) => setSelectedDerivationScheme(e)}
+        >
+          {Object.keys(derivationPathsEnum).map((path) => {
+            const pathValue = derivationPathsEnum[path]
+            const pathLocale =
+              HardwareWalletDerivationPathLocaleMapping[pathValue]
+
+            return <div data-value={pathValue}>{pathLocale}</div>
+          })}
+        </Select>
+      </SelectWrapper>
+      <DisclaimerWrapper>
+        <DisclaimerText>{locale.switchHDPathTextHardwareWallet}</DisclaimerText>
+      </DisclaimerWrapper>
+      <SearchBar placeholder={locale.searchScannedAccounts} action={() => {}} />
```

Comment:
Can we add a `filter` method here to pass into the `action` prop to give the ability to `search`